packages/slice-machine/components/AppLayout/Navigation/Menu/Navigation/VideoItem.tsx

```diff
@@ -25,11 +23,7 @@ const VideoItem: FC<VideoItemProps> = ({
   const videoUrl = VIDEO_YOUTUBE_PLAYLIST_LINK;
 
   const handleClose = () => {
-    void Tracker.get().trackClickOnVideoTutorials(
-      framework,
-      sliceMachineVersion,
```

Comment:
Is Come ok with this change?

Reply:
...wait looks like I deleted to much there :)

Reply:
no this is correct, the version is added on the `/s` route